deployment/roles/deploy/templates/centos-secrets.yml.j2

```
@@ -0,0 +1,12 @@
+# Copyright Contributors to the Packit project.
+# SPDX-License-Identifier: MIT
+
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: centos-secrets
+type: Opaque
+data:
+  centos-server-ca.cert: "{{ lookup('file', '{{ path_to_secrets }}/centos-server-ca.cert') | b64encode }}"
+  centos.cert: "{{ lookup('file', '{{ path_to_secrets }}/centos.cert') | b64encode }}"
```

I'd start simple, with the path to secrets hardcoded.

```suggestion
  centos-server-ca.cert: "{{ lookup('file', 'secrets/centos-server-ca.cert') | b64encode }}"
  centos.cert: "{{ lookup('file', 'secrets/centos.cert') | b64encode }}"
```

I tried locally and it works if you put those `*.cert` into `roles/deploy/files/secrets/`.

---

I've created #87 which you can build upon.